<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>제주고딕</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#00aa00;&quot;&gt;네이버지도 맛집 검색결과 엑셀저장&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <family>제주고딕</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>검색어</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="keyword">
             <property name="font">
              <font>
               <family>제주고딕</family>
               <pointsize>12</pointsize>
              </font>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <family>제주고딕</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>맛집 개수</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="place_max">
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::UpDownArrows</enum>
             </property>
             <property name="specialValueText">
              <string>최대로</string>
             </property>
             <property name="correctionMode">
              <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
             </property>
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="start_btn">
           <property name="font">
            <font>
             <family>제주고딕</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>시작</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="remove_btn">
           <property name="text">
            <string>저장위치 초기화</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cur_location_btn">
           <property name="text">
            <string>현재 저장 위치</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextBrowser" name="status">
         <property name="font">
          <font>
           <family>제주고딕</family>
           <pointsize>8</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="location_btn">
         <property name="font">
          <font>
           <family>제주고딕</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>저장위치 설정</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="reset_btn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>제주고딕</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>리셋</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="quit_btn">
         <property name="font">
          <font>
           <family>제주고딕</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>종료</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
